app/config/ApplicationConfig.scala

```diff
@@ -94,6 +94,11 @@ class ApplicationConfig @Inject()(config: Configuration, runMode: RunMode) exten
   val ppnsSandboxApiKey = getConfString("push-pull-notifications-api-sandbox.api-key", "")
   val ppnsSandboxAuthorizationKey = getConfString("push-pull-notifications-api-sandbox.authorizationKey", "")
 
+  // Fraud Prevention Headers
+  lazy val fraudPreventionLinkVisible = getConfigDefaulted("fraudPreventionLinkVisible", false)
+  lazy val fraudPreventionApis = getConfigDefaulted[Seq[String]]("fraudPreventionApis", List.empty)
```

Comment:
And can we put this in a separate config item and add the provider for it.  We're also trying to break down large config.